tests/test_utils.py

```diff
@@ -362,29 +362,29 @@ def test_augment_diagonal_undirected(self):
         )
         expected = A.copy().astype(float)
         expected[0, 0] = 2.0 / 4
-        expected[1, 1] = 3.0 / 4
+        expected[1, 1] = 4.0 / 4
```

Comment:
why are you changing anything in this file

Reply:
I didn't change them manually in any way. I cloned it from the very master branch but I don't know why these difference happens

Reply:
okay, fix it